ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -1901,6 +1901,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
Please note this `always_run: false` here. I'm doing it intentionally because of a limited vSphere dual-stack cluster availability. I'm open to changing it to `always_run: true` to run always, but the impact may be that PRs are slower to merge

Reply:
w/ always_run and optional both set to false, this job will never run on it's own. that's fine, but unless folks know
and care about it, it will never be run. Initially, this might be what we want and those concerned or working
in this dualstack/vsphere area will trigger it and it will serve it's purpose.

However, if the purpose of this job is to prevent regressions, then we can do a couple of things:

1) make it required (optional and always_run both are true) and it will block PRs from merging.
2) make it a periodic job and ensure someone is monitoring

if the job is not passing at a high rate (90%+) then I would not suggest 1)

Reply:
I was trying to avoid a scenario in which we have this job slowing down the process of PR merge too much. But yeah, without `always_run` people will forget about it soon. I have changed it to be optional but always run, that should be a fair trade off